examples/freertos_plus_tcp/z_put.c

```diff
@@ -67,7 +67,7 @@ void app_main(void) {
 
     // Create payload
     z_owned_bytes_t payload;
-    z_bytes_serialize_from_str(&payload, VALUE);
+    z_bytes_from_str(&payload, (char *)VALUE, NULL, NULL);
```

Comment:
I believe we can accept `const char*` here. This is very typical case when we need to send string without providing destructor (static one or allocated on command line parsing. Also payload is not supposed to be able to change the content of the string, it only takes ownership on it. So I see no reason to force user to write `(char*)`every time

Reply:
It's a good point that "free" can't be called on const pointer.
Then it makes even more sense to have 2 functions:
```
z_bytes_from_static_str(z_owned_payload_t* payload, const char* str);
z_bytes_from_str(z_owned_payload_t* payload, char* str,  void (*deleter)(void *data, void *context),
 void *context);
```
and maybe even disallow to call second one with empty destructor

And do the same for sized buffer of course

Reply:
I prefer to be explicit in this case and piggy-backing on any warning the compiler would give about discarding any `const` qualifier.